scripts/submit_csv.sh

```diff
@@ -72,11 +77,28 @@ sed "
 " templates/${TEMPLATE}.submit.in > ${SUBMIT_FILE}
 
 # submit job
-condor_submit -verbose -file ${SUBMIT_FILE}
+if [[ ${TEMPLATE} =~ slurm ]] ; then
 
-# create log dir
-if [ $? -eq 0 ] ; then
-  for i in `condor_q --batch | grep ^${USER} | tail -n1 | awk '{print($NF)}' | cut -d. -f1` ; do
-    mkdir -p LOG/CONDOR/osg_$i/
+  # slurm
+  max_array_size=2000
```

Comment:
Site dependent. No general way to query this on slurm clusters. Assumed but can also grep from /etc/slurm/slurm.conf, which is typical location. Or fall back to default of 1000.